src/CMakeLists.txt

```diff
@@ -220,8 +221,7 @@ target_sources(
     WrapCalls.h
 )
 
-# The sources that go into libHalide. For the sake of IDE support, headers that
-# exist in src/ but are not public should be included here.
+# The sources that go into libHalide.
```

Comment:
Why did you change this comment?

Reply:
Hehehehehe: https://github.com/halide/Halide/pull/8388#discussion_r1848727520

Reply:
Ay yi yi -- sorry I forgot. I'm guessing there are some headers that _ought_ to be in that list.